README.rst

```diff
@@ -47,7 +58,7 @@ using mlschema-model-converters_ plugin.
 
     export(model, evaluation_measure=(accuracy_score, acc))
 
-Using explicitly the `export` function the details of the supplied model are going to 
+Using explicitly the `export` function the details of the supplied model are going to
```

Comment:
This sentence does not read very well.

Reply:
thanks for flagging it - I tried to improve it